src/node_env_var.cc

```diff
@@ -305,8 +273,13 @@ static void EnvGetter(Local<Name> property,
     return info.GetReturnValue().SetUndefined();
   }
   CHECK(property->IsString());
-  info.GetReturnValue().Set(
-      env->env_vars()->Get(env->isolate(), property.As<String>()));
+  MaybeLocal<String> value_string =
+      env->env_vars()->Get(env->isolate(), property.As<String>());
+  if (value_string.IsEmpty()) {
```

Comment:
You can just do

```
if (!value_string.IsEmpty()) {
  info.GetReturnValue().Set(value_string.ToLocalChecked());
}
```

here.

Reply:
Changes are committed at https://github.com/nodejs/node/pull/27310/commits/34c8538aef50d3a392dc4d6522c472f57cc2472e